scripts/jenkins/msvc.groovy

```diff
@@ -26,6 +26,7 @@ node('win && conan') {
 
         stage 'Test (Win)'
         build.win 'build', 'tests'
+        build.win 'build', 'ctest'
```

Comment:
Yes [it is release](https://github.com/ufz/ogs/blob/master/scripts/jenkins/lib/build.groovy#L24).